spec/preload.lua

```diff
@@ -3,6 +3,9 @@
 -- is not safe to reload and yet Busted manages to do this.
 require("lgi")
 
+-- Prevent deprecated messages from showing in the Travis output
```

Comment:
Well, it's the opposite, isn't it? This does *force* all deprecation messages to show up. However the line itself is necessary. If it's not there, then there is build errors as `capi.awesome` is `nil`. I would chnage the comment to

> -- Display deprecated messages even for pending deprecation